setup.py

```diff
@@ -24,7 +24,7 @@
         "pyaudio",
         "webrtcvad",
         "numpy",
-        "websocket",
+        "websocket_client",
```

Comment:
this should at least be a different commit, probably a separate pr

Reply:
yeah this was an accident. we will make the change though because `websocket_client` is the one we use. there are a couple with similar names which is annoying.